<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxEditorClass</class>
 <widget class="QMainWindow" name="GfxEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gfx Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="actionClone"/>
   <addaction name="separator"/>
   <addaction name="actionObjectSelect"/>
   <addaction name="actionSelectByName"/>
   <addaction name="separator"/>
   <addaction name="actionObjectMove"/>
   <addaction name="actionObjectRotate"/>
   <addaction name="actionObjectScale"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="sideBar">
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>38</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Side Bar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New Gfx</string>
   </property>
   <property name="toolTip">
    <string>Create a effect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Gfx...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Gfx</string>
   </property>
   <property name="toolTip">
    <string>Save file to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>CloseClose this window</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionEditUndo">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete(Del)</string>
   </property>
   <property name="statusTip">
    <string>Delete selected objects</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
   <property name="toolTip">
    <string>Clone(Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionObjectSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/select.png</normaloff>:/images/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select(Q)</string>
   </property>
   <property name="statusTip">
    <string>Select Object</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionObjectRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/rotate.png</normaloff>:/images/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate(R)</string>
   </property>
   <property name="statusTip">
    <string>Rotate Object</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionObjectMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/move.png</normaloff>:/images/move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move(E)</string>
   </property>
   <property name="statusTip">
    <string>Move Object</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionObjectScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/scale.png</normaloff>:/images/scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="toolTip">
    <string>Scale(T)</string>
   </property>
   <property name="statusTip">
    <string>Scale Object</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save Map As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCreateTerrain">
   <property name="text">
    <string>Create Terrain</string>
   </property>
  </action>
  <action name="actionUseSelectionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/gizmo_selection_center.png</normaloff>:/images/gizmo_selection_center.png</iconset>
   </property>
   <property name="text">
    <string>Use Selection Center</string>
   </property>
  </action>
  <action name="actionUsePivotCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/gizmo_pivot_center.png</normaloff>:/images/gizmo_pivot_center.png</iconset>
   </property>
   <property name="text">
    <string>Use Pivot Center</string>
   </property>
  </action>
  <action name="actionUseTransformCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/gizmo_transform_center.png</normaloff>:/images/gizmo_transform_center.png</iconset>
   </property>
   <property name="text">
    <string>Use Transform Center</string>
   </property>
  </action>
  <action name="actionSnapToGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/snap_to_grid.png</normaloff>:/images/snap_to_grid.png</iconset>
   </property>
   <property name="text">
    <string>Snap To Grid</string>
   </property>
  </action>
  <action name="actionSnapToAngle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/snap_to_angle.png</normaloff>:/images/snap_to_angle.png</iconset>
   </property>
   <property name="text">
    <string>Snap To Angle</string>
   </property>
  </action>
  <action name="actionSelectByName">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/select_by_name.png</normaloff>:/images/select_by_name.png</iconset>
   </property>
   <property name="text">
    <string>Select By Name</string>
   </property>
  </action>
  <action name="actionLayer">
   <property name="icon">
    <iconset resource="resources/editor.qrc">
     <normaloff>:/images/layer.png</normaloff>:/images/layer.png</iconset>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSelectInvert">
   <property name="text">
    <string>Select Invert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
